fix: replace hash icon to arrow down for filter select

Gourav2609 opened the pull request:
## Description

i have done the changes suggested by @BekahHW  in PR #1816  
and issue #1812 

Please review this PR



# Before

![273069319-c8786858-b1c2-4430-ae1b-172eb5974d8c](https://github.com/open-sauced/app/assets/115522947/94789cce-d64a-4fa3-85ac-7797266da52d)

# After

![image](https://github.com/open-sauced/app/assets/115522947/6535db6a-b4e2-4ce5-9859-0346f935ee7e)


<!-- 
Please do not leave this blank 
This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 
-->

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [1 ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
<!-- 
Please use this format link issue numbers: Fixes #1812 
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

## Mobile & Desktop Screenshots/Recordings


https://github.com/open-sauced/app/assets/115522947/59e9f461-5600-420b-bb92-eef7a556d3ca




<!-- Visual changes require screenshots -->


## Added tests?

- [ ] 👍 yes
- [ 1] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

### NO THERE IS NO NEED FOR THAT


## [optional] What gif best describes this PR or how it makes you feel?



<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
  
  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

Gourav2609 commented:
@BekahHW  any suggestions?

bdougie commented on components/molecules/FilterCardSelect/filter-card-select.tsx:
```diff
@@ -4,10 +4,11 @@ import Image from "next/image";
 
 import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "components/atoms/Select/select";
-import hashIcon from "../../../img/icons/hash.svg";
+// import hashIcon from "../../../img/icons/hash.svg";
```
dead code needs to be removed.

Gourav2609 replied:
@bdougie Please checkout the latest commit . 
i have remove the comment code and fixed the hash.svg ....
Earlier i was using a different svg code from ....but now i have fixed it with the same code as it was before

bdougie reviewed (dismissed):
Where are you sourcing the SVG? The # looks rather big.

bdougie commented:
Mobile screen doesn't have the arrow pointing down.  ![image](https://github.com/open-sauced/app/assets/5713670/1d33bc94-a02b-48ce-822c-debfc5657904)

jpmcb reviewed (commented):
This shows up as a left facing arrow on Safari mobile: is that intended?

![image](https://github.com/open-sauced/app/assets/23109390/3dc15123-8c1b-4fdb-8430-bc70acfec56e)

Gourav2609 commented:
> Mobile screen doesn't have the arrow pointing down. ![image](https://user-images.githubusercontent.com/5713670/273679224-1d33bc94-a02b-48ce-822c-debfc5657904.jpeg)

I tried to replicate the error but could not do so
![image](https://github.com/open-sauced/app/assets/115522947/6b8a868d-e3bf-4d85-a1d2-0613b9eccc6f)

jpmcb commented:
![Screenshot 2023-10-11 at 8 41 02 AM](https://github.com/open-sauced/app/assets/23109390/5e9d64c5-fdb8-44be-bd3f-bb2db7dc9147)

It must be something with how Safari renders these icons. Here is Safari desktop with the same problem.

nickytonline commented:
I believe we want to be using this icon, https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg @Gourav2609

Gourav2609 commented:
> I believe we want to be using this icon, https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg @Gourav2609

sure ....i will revert the changes and look for a solution

Gourav2609 commented:
@bdougie @jpmcb i have replaced svg with png. this should fix the safari error

nickytonline commented on img/icons/chevron-left.svg:
Refactor (blocking): These icons should not be part of the pull request. The icon required for the fix is already in the project.

https://github.com/open-sauced/app/blob/beta/img/chevron-down.svg

Gourav2609 replied:
I have removed inverted-triangle.png and currently using chevron-down.svg that was already there in the img .

OgDev-01 commented:
Looks good on my end(chrome)... i'll leave it up to @nickytonline and @jpmcb to drop their review based on browser support 👍
![image](https://github.com/open-sauced/app/assets/62995161/6132f545-9f88-40bc-b20b-e94ca224c85d)

Gourav2609 commented:
@OgDev-01 Thanks for the review

i would also request @nickytonline @bdougie do suggest any changes

Thank in advance

nickytonline reviewed (changes requested):
Refactor (blocking): There's changes to the chevron-left icon which shouldn't be a part of this PR.

I'll take it for a spin, but once the above icon changes are removed, this should be good.

Gourav2609 commented:
> Refactor (blocking): There's changes to the chevron-left icon which shouldn't be a part of this PR.
> 
> I'll take it for a spin, but once the above icon changes are removed, this should be good.

i changed the code for already present `chevron-left.svg` in my past commits trying different ways....in my most recent commit `fixed icons` ... i have added no new icons or svgs...used the already present `chevron-down.svg `

Do i still need to do any changes...Please let me know @bdougie @nickytonline @OgDev-01

nickytonline commented:
@Gourav2609, the chevron-left icon changes are still in the pull request. What I suggested won't work because your pull request is using the beta branch in your fork.

You'll need to revert it instead to an a version prior to your changes, e.g.

1. `git checkout 4d240a84 -- img/icons/chevron-left.svg`
1. `git commit -m "chore: reverted changes to chevron-left icon"`
1. `git push origin your_branch_name`

Gourav2609 commented:
@nickytonline Thanks so much for the help

Can u please review it

nickytonline reviewed (dismissed):
Thanks @Gourav2609. Looks good on Chromium, Firefox & Safari

**Chromium**

![CleanShot 2023-10-16 at 16 47 13](https://github.com/open-sauced/app/assets/833231/53e262f5-9ff1-44a7-bd56-13e566aed292)

**Firefox**

![CleanShot 2023-10-16 at 16 48 28](https://github.com/open-sauced/app/assets/833231/8cd4c23b-492a-424a-939c-96db54a7d1c7)

**Safari**

![CleanShot 2023-10-16 at 16 49 14](https://github.com/open-sauced/app/assets/833231/986780aa-8b8a-4e05-8a5b-e0a81f5ba57c)

Gourav2609 commented:
Thanks @nickytonline 

any suggestions from @bdougie @jpmcb

nickytonline commented:
> Thanks @Gourav2609. Looks good on Chromium, Firefox & Safari

Looking at the icon placement actually, it could be closer to the right. That might be unrelated to your change though. Just taking a peek.

Gourav2609 commented:
In issue #1812 , @bdougie pointed about the icon being slightly towards right and wanted it to be aligned properly...in response to that i aligned it to center...
No prob if you want i can shift it towards right🙂

nickytonline reviewed (approved):
Alright, let's go ahead with this. Maybe going forward we use your new dropdown component @OgDev-01?

OgDev-01 commented:
> Maybe going forward we use your new dropdown component @OgDev-01?

Ofcourse 👍. we can merge this as is

OgDev-01 reviewed (approved):
🍕

Gourav2609 commented:
thanks @bdougie @nickytonline @OgDev-01  and everyone...looking forward to contribute more😊
